apps/mobile/src/queries/balance/bitcoin-balance.query.ts

```diff
@@ -66,11 +68,13 @@ export function useTotalBitcoinBalanceOfDescriptors(descriptors: string[]) {
       );
 
       const availableBalance = createMoney(amount, 'BTC');
-      const fiatBalance = calcUsdAmount(availableBalance);
+      const fiatBalance = btcMarketData
+        ? baseCurrencyAmountInQuote(availableBalance, btcMarketData)
+        : createMoney(0, 'USD');
```

Comment:
Not for this PR but we have to use this `createMoney(0, 'USD')` fallback sometimes which is technically not correct to show a `0` balance. 

We need to get [this issue](https://linear.app/leather-io/issue/LEA-505/loading-state-for-balances-on-home-screen) decided upon soon so we can show something instead of `0` when the service can't get the balance cc @camerow